src/test/java/com/mentoring/test/FriendRequestTest.java

```diff
@@ -0,0 +1,104 @@
+package com.mentoring.test;
+
+import com.mentoring.framework.BasicTest;
+import com.mentoring.framework.database.Database;
+import com.mentoring.framework.database.DbSelects;
+import com.mentoring.framework.utils.UserUtils;
+import com.mentoring.model.User;
+import com.mentoring.pageobject.*;
+import com.mentoring.test.assertionsteps.CommonAssertions;
+import com.mentoring.util.Messages;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.*;
+
+import java.sql.SQLException;
+
+import static com.mentoring.model.Features.FRIEND_REQUEST;
+
+@Slf4j
+public class FriendRequestTest extends BasicTest {
+
+    private SoftAssertions softAssertion;
+    private Database database;
+    private User userA;
+    private User userB;
+    private String characterNameA;
+    private String characterNameB;
+    String accessTokenIdForUserA;
+    String accessTokenIdForUserB;
+    String userIdForUserA;
+    String userIdForUserB;
+
+    @BeforeClass(alwaysRun = true)
+    private void setup() {
+        database = new Database();
+    }
+
+    @AfterClass(alwaysRun = true)
+    private void close() {
+        database.disconnectFromDb();
+    }
+
+    @BeforeMethod(alwaysRun = true)
+    private void initMethod() throws SQLException {
```

Comment:
setup() would be more staightforward and shorter method name I believe. Just a suggestion.